verilog/CST/declaration.h

```diff
@@ -132,6 +134,12 @@ const verible::TokenInfo& GetModuleInstanceNameTokenInfoFromGateInstance(
 const verible::TokenInfo& GetInstanceNameTokenInfoFromRegisterVariable(
     const verible::Symbol&);
 
+// For a given node tagged with kVariableDeclarationAssign return the
+// unqualified id inside that node.
```

Comment:
there can be many unqualified ids.  can you provide an example?  from "x" extracts "y"?